index.js

```diff
@@ -258,8 +258,12 @@ module.exports = function (options) {
         return next(e);
       }
 
-      // validate status
-      if (parsedResponse.status !== 'urn:oasis:names:tc:SAML:2.0:status:Success') {
+      // validate statuses
+      const validStatuses = [
+        'urn:oasis:names:tc:SAML:2.0:status:Success', 
+        'urn:oasis:names:tc:SAML:2.0:status:PartialLogout'
+      ];
+      if (!validStatuses.includes(parsedResponse.status)) {
```

Comment:
By spec `urn:oasis:names:tc:SAML:2.0:status:PartialLogout` is only allowed to exist as second-level status code (https://docs.oasis-open.org/security/saml/v2.0/saml-core-2.0-os.pdf Section 3.2.2.2). So that we would expect it in `parsedResponse.subCode`.

I took a look to node-samlp repo, and maybe this need is related with which seems a bug a there, it sets the partial logout as top-level status code. 
https://github.com/auth0/node-samlp/blob/master/lib/logout.js#L75

Reply:
Perfect!